artifacts/openapi.yaml

```diff
@@ -3528,6 +3532,527 @@ components:
+      properties:
+        setup_priority:
+          description: |-
+            Specifies the value of the Setup Priority field. This controls whether the backup LSP should pre-empt existing LSP that is setup with certain Holding Priority. While setting up a backup LSP, preemption of existing LSP can happen  if resource limitation is encountered (e.g bandwidth availability).
+          type: integer
+          default: 7
+          minimum: 0
+          maximum: 7
+        holding_priority:
+          description: "Specifies the value of the Holding Priority field. This controls\
+            \ whether a new LSP being created with certain Setup Priority should pre-empt\
+            \ this LSP set up with this Holding Priority. While setting up a new LSP,\
+            \ preemption of existing LSP can happen  if resource limitation is encountered\
+            \ (e.g bandwidth availability).            "
+          type: integer
+          default: 7
+          minimum: 0
+          maximum: 7
+        hop_limit:
+          description: |-
+            Specifies the value of the Hop Limit field. This controls the maximum number of hops the LSP should traverse to reach the  LSP end-point.
+          type: integer
+          default: 3
+          minimum: 0
+          maximum: 255
+        bandwidth:
+          description: "Specifies the value of the Bandwidth field in bytes per second\
+            \ desired for the LSP. "
+          type: integer
+          format: int64
+          default: 0
+          minimum: 0
+          maximum: 4294967295
+        exclude_any:
+          description: "A 32-bit vector representing a set of attribute filters associated\
+            \ with a tunnel any of which renders a link unacceptable.             "
+          type: string
+          format: hex
+          default: '0'
+        include_any:
+          description: "A 32-bit vector representing a set of attribute filters associated\
+            \ with a tunnel any of which renders a link acceptable.             "
+          type: string
+          format: hex
+          default: '0'
+        include_all:
+          description: "A 32-bit vector representing a set of attribute filters associated\
+            \ with a tunnel all of which must be present for a link to be acceptable.\
+            \            "
+          type: string
+          format: hex
+          default: '0'
+        one_to_one_backup_desired:
+          description: "Requests protection via the one-to-one backup method.    \
+            \       "
+          type: boolean
+          default: false
+        facility_backup_desired:
+          description: "Requests protection via the facility backup method.      \
+            \     "
+          type: boolean
+          default: false
+    Rsvp.Ero:
+      description: "Configuration for the optional RSVP-TE ERO object included in\
+        \ Path Messages. "
+      type: object
+      properties:
+        prepend_egress_ip:
```

Comment:
I think it should be prepend_dut_ip or prepend_neighbor_ip

Reply:
Good catch. Done.